workflow/envs/fastx_toolkit.yaml

```diff
@@ -0,0 +1,7 @@
+---
+channels:
+  - conda-forge
+  - kantorlab
```

Comment:
I used this channel because it was the only one in `Anaconda` that provides the same names for the reads in the `FASTA` transformation otherwise I had to change the expected output file.

I can change it to use the `bioconda` channel but that would imply changing the expected output.